packages/editor-ui/src/components/WorkflowPreview.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+	<iframe
+		:class="{ workflow_iframe: !this.nodeView, workflow_iframe_node_view: this.nodeView }"
+		id="preview_iframe"
+		href="/workflows/demo"
+	></iframe>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({
```

Comment:
also add component name
```suggestion
export default Vue.extend({
   name: 'WorkflowPreview',
```